### PR TITLE
[#128143135] Feature: Barcodes support for Carter

### PR DESCRIPTION
**Pull request by Tonkonozhenko**

### CODE REVIEW

File: source/campaigns/editor/filters.rst [hunk 1]
```diff
@@ -504,4 +504,17 @@ Refresh the screenshot every week:
...
+barcode
+.......
+
+Generates data for displaying barcode according to format GS1-128.
+
+.. code-block:: liquid
+
+    {% assign barcode = "X" | barcode %}
+
+Returns array of booleans: ``[true, true, false, ...]``.
```
Comment (iurevych) [inline source/campaigns/editor/filters.rst]:
``` rst
barcode
.......

Represents any ``string`` as an array of boolean values: ``[true, false]`` in order to convert it into a barcode. This filter strictly follows GS1-128 specification: https://en.wikipedia.org/wiki/GS1-128.

Here is an example how to convert a coupon code into a barcode:

  CODE GOES HERE, PLEASE PROVIDE A VALID CODE SAMPLE

File: source/campaigns/editor/filters.rst [hunk 2]
```diff
@@ -504,4 +504,45 @@ Refresh the screenshot every week:
...
+
+**Example usage in template**
+
+HTML:
+
+.. code-block:: html
+
+   <table cellspacing="0" cellpadding="0" border="0">
```
Comment (iurevych) [inline source/campaigns/editor/filters.rst]: Add a check for blank:
``` liquid
{% if coupon_code != blank %}
```

Comment (Tonkonozhenko) [inline source/campaigns/editor/filters.rst]: I think we don't need it. This code must be placed in that `if` branch where coupon_code is present

**Comment by iurevych** [comment]
Looks good

**Pull request merged**